_ci/verify-release-assets.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Script to verify the release assets by downloading them from the GitHub release
```

Comment:
You might want to `set -euo pipefail` so that the script exits early if something unexpected fails.

Reply:
Added